lib/jekyll/tags/link.rb

```diff
@@ -18,14 +18,17 @@ def initialize(tag_name, relative_path, tokens)
       def render(context)
         site = context.registers[:site]
 
+        liquid = site.liquid_renderer.file("(jekyll:link)")
```

Comment:
@alexey-pelykh can you please explain what `"(jekyll:link)"` is supposed to mean here? That is beyond my Ruby knowledge..